### ISSUE TITLE
[DatePicker] Background highlight on today's date not visible in high contrast mode in Chromium Edge

### ISSUE THREAD

**Issue by stacymiller96** [microsoft/fluentui#12444]
### Environment Information

- **Package version(s)**:  Observable in version used in our product (v6.211.2) and current version 7 used in Fabric documentation website/codepen below.
- **Browser and OS versions**: Edge (Chromium)

### Describe the issue:
In Edge (Chromium), using DatePicker in high contrast mode, the today date in the month view is not highlighted as it is when using in legacy Edge. 

### Please provide a reproduction of the issue in a codepen: https://codepen.io/stmil/pen/RwPqqvy
(Use HCM in new Edge and compare against legacy Edge)

#### Actual behavior:

Today date in month picker is not highlighted in high contrast mode in Edge Chromium.

Example screenshots of issue (Today is March 26):

Edge (Legacy):
![image](https://user-images.githubusercontent.com/15148186/77689459-e9e61380-6f5e-11ea-82b0-4c3a33f1cec9.png)

Edge (New):
![image](https://user-images.githubusercontent.com/15148186/77689552-1306a400-6f5f-11ea-8bd5-81565a5ab1eb.png)

#### Expected behavior:

Today date in month picker is highlighted in high contrast mode in Edge Chromium, the same as in Edge Legacy.

**Comment by aneeshack4** [comment]
@stacymiller96 Thanks for reporting this! I repro this on both High Contrast White and Black. There are known high contrast support issues on Chromium #11742 so this might be related to those. Will look into this more and report back.
